sdk/search/Microsoft.Azure.Search/src/Microsoft.Azure.Search.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Makes it easy to develop a .NET application that uses Azure Search.</Description>
     <AssemblyTitle>Microsoft Azure Search Library</AssemblyTitle>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-    <PackageReleaseNotes>This is the newest major version of the Azure Search .NET SDK, based on version 2019-05-06 of the Azure Search REST API. New in this GA version is support for Cognitive Search, complex types for richer data modeling, the Autocomplete API, and additional parsing modes for Blob indexers. See this article for help on migrating to the latest version: http://aka.ms/search-sdk-upgrade. The 9.0.1 release includes a fix for the deadlock issue (https://github.com/Azure/azure-sdk-for-net/issues/6254) in the Search API.</PackageReleaseNotes>
+    <PackageReleaseNotes>This is the newest major version of the Azure Search .NET SDK, based on version 2019-05-06 of the Azure Search REST API. New in this GA version are a few new skills and some general improvements that necessitate breaking changes. See this article for help on migrating to the latest version: http://aka.ms/search-sdk-upgrade. Complete release notes for this package are located here: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/search/Microsoft.Azure.Search/changelog.md.</PackageReleaseNotes>
```

Comment:
Assuming the ChangeLog is ever growing and never pruned then having a link to master could work, assuming the location of the file doesn't move over time, but it might be better to use a commit sha or tag instead of master in your link.

Reply:
That's a reasonable point - to keep with the spirit, let me create another aka.ms link and point it to the master... that way, any changes to the location of the changelog wouldn't require a change to the csproj itself and it'll be similar to the SDK upgrade guide

Reply:
While you could use an aka.ms link I'm not sure if you want to maintain that just for this. Even in that case it would always be pointing to the latest changelog which may or may not be what you want. If you want the change log to represent exactly what the source was when you shipped then you could use the sha.